scripts/tiqit/__init__.py

```diff
@@ -998,7 +998,7 @@ def getBasePath():
     return basepath
 
 def getBaseHost():
-    return 'http://%s%s' % (os.environ['HTTP_HOST'], getBasePath())
+    return 'https://%s%s' % (os.environ['HTTP_HOST'], getBasePath())
```

Comment:
Some of the link styles in static/styles/tiqit.css might need an update to avoid breaking them.